### PR TITLE
Stack convert to zemax

### PR DESCRIPTION
**Pull request by mcheng2022**
Not sure if I'm correctly creating this pull request. Please let me know if I did it wrong.
Here is code to test in terminal. Change the file path to yours.

from ansys_optical_automation.post_process.dpf_stack import DpfStack
stack_dpf = DpfStack(222)
stack_dpf.open_file(r'C:\Users\mcheng\OneDrive - ANSYS, Inc\SampleFrCust\test_10_stack_interop.ldf')
stack_dpf.convert_stack_to_speos()
stack_dpf.convert_stack_to_zemax()

### CODE REVIEW

**Comment by pluAtAnsys** [comment]
As suggested by the failed pre-commmit style automation check, please run "pre-commit run -a" at terminal to make sure following Python PEP 8 standard.
In addition, as you have added new function/method to the library, you also need to create unit test cases to make sure your function is covered by test use case, e.g. test to make sure a *.dat file is generated + test to make sure the generated *.dat file is generated with correct content.

**Comment by mcheng2022** [comment]
@pluAtAnsys Do you think I can just edit the test_10 you build for the unit test? Or should I create a new one? Thank you.

**Comment by pluAtAnsys** [comment]
> @pluAtAnsys Do you think I can just edit the test_10 you build for the unit test? Or should I create a new one? Thank you.

yep, please use the test_10 as that is test group for stack related interop.

**Comment by mcheng2022** [comment]
> > @pluAtAnsys Do you think I can just edit the test_10 you build for the unit test? Or should I create a new one? Thank you.
> 
> yep, please use the test_10 as that is test group for stack related interop.

Added!

File: tests/test_10_stack_interop.py [hunk 1]
```diff
@@ -37,28 +45,55 @@ def clean_results(self):
...
-        ref_file = open(self.reference_file, "r")
+        res_file = open(self.results_file_speos, "r")
+        ref_file = open(self.reference_file_speos, "r")
+        res = res_file.read()
+        ref = ref_file.read()
+        res_file.close()
+        ref_file.close()
+        assert res == ref
+
+    def test_03_stack_speos_generated(self):
```
Comment (pluAtAnsys) [inline tests/test_10_stack_interop.py:78]:
```suggestion
    def test_03_stack_zemax_generated(self):
```

**Review by pluAtAnsys — CHANGES_REQUESTED** [review]

**Review by pluAtAnsys — APPROVED** [review]

**Pull request merged**